<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-O2V6U4O" ignored="1" name="Cart Test" tests="3" failures="3" timestamp="2020-05-23T09:50:47 BST" time="0.036" errors="0">
  <testcase name="assertionNotEqualsTest" time="0.001" classname="testngtests.AssertionsExamplesTests">
    <failure type="java.lang.AssertionError" message="I am failure Message : Titles should not match biut they are matching.  did not expect [Something] but found [Something]">
      <![CDATA[java.lang.AssertionError: I am failure Message : Titles should not match biut they are matching.  did not expect [Something] but found [Something]
at testngtests.AssertionsExamplesTests.assertionNotEqualsTest(AssertionsExamplesTests.java:53)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- assertionNotEqualsTest -->
  <testcase name="additionToCartTest" time="0.001" classname="testngtests.SampleTest1">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
at testngtests.SampleTest1.additionToCartTest(SampleTest1.java:27)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- additionToCartTest -->
  <testcase name="deletingFromCartTest" time="0.011" classname="testngtests.SampleTest1">
    <failure type="java.lang.AssertionError" message="expected [0] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [3]
at testngtests.SampleTest1.deletingFromCartTest(SampleTest1.java:46)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deletingFromCartTest -->
  <testcase name="assertionEqualsTest" time="0.0" classname="testngtests.AssertionsExamplesTests">
    <ignored/>
  </testcase> <!-- assertionEqualsTest -->
</testsuite> <!-- Cart Test -->
